src/main/webapp/WEB-INF/components/header.html

```diff
@@ -27,7 +27,12 @@
         <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded" href="/idk">idk</a></li>
       </ul>
       <ul class="navbar-nav">
-        <li class="nav-item mx-0 mx-lg-1"><a class="nav-link py-3 px-0 px-lg-3 rounded" href="/login">Login</a></li>
+        <li id="login-status" class="nav-item mx-0 mx-lg-1">
+          <a id="login-status-link" class="nav-link py-3 px-0 px-lg-3 rounded" href="#"></a>
+        </li>
+        <li id="login-status" class="nav-item mx-0 mx-lg-1">
+          <a id="header-username" class="nav-link py-3 px-0 px-lg-3 rounded" href="#"></a>
+        </li>
```

Comment:
Why are there two nodes with the same id (`login-status`)? Maybe it should be a class?

Reply:
+1

Reply:
Sorry, I deleted the ids, since they are not used anywhere.